crates/apub/src/inbox/person_inbox.rs

```diff
@@ -25,7 +25,7 @@ use crate::{
     inbox_verify_http_signature,
```

Comment:
We should probably rename `community_inbox.rs` -> `group_inbox.rs` as well, otherwise its weird and inconsistent.

Reply:
Lets cross that bridge when / if we eventually split the community table into federated and local too. I remember it wasn't really worth doing because there were only like 1-2 local only columns.

Reply:
Not necessary, its just weird to have files `person_inbox.rs` and `community_inbox.rs` next to each other.